lib/routes.jsx

```diff
@@ -29,6 +29,7 @@ var pages = {
   'clubs/list': require('../pages/clubs-list.jsx'),
   'community': require('../pages/community.jsx'),
   'community/curriculum-workshop': require('../pages/curriculum-workshop.jsx'),
+  'community/curriculum-workshop-march-8-2016': require('../pages/curriculum-workshop-march-8-2016.jsx'),
```

Comment:
this url will link specifically to the archived workshop?

I don't have a strong opinion (and I believe @chadsansing is on PTO), but does it make more sense to use community/curriculum-workshop/march-8-2016?